modules/upgrade-duration-mco.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding-openshift-upgrade-duration.adoc
+
+:_content-type: CONCEPT
+[id="machine-config-operator-node-updates_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and worker node. During this process, the MCO performs the following sequential actions on each node of the cluster:
+
+. Cordon and drain all the nodes
+. Update the operating system (OS)
+. Reboot the nodes
+. Uncordon all nodes and schedule workloads on the node
+
+[NOTE]
+====
+When a node is cordoned, it is unavailable to schedule a workload.
+====
+
+The time to complete this process depends on several factors including the node and infrastructure configuration. This process might take 5 or more minutes to complete per node.
+
+In addition to MCO, you should consider the impact of the following parameters:
+
+* The control plane node upgrade duration is predictable and oftentimes shorter than worker nodes, because the control plane workloads are tuned for graceful updates and quick drains.
+
+* You can upgrade the worker nodes in parallel by setting the `maxUnavailable` field to greater than 1 in the machine config pool. The MCO cordons the number of nodes specified in `maxUnavailable` and marks them unavailable for upgrade.
+
+* When you increase `maxUnavailable` on the machine config pool, it can help the pool to update more quickly. However, if `maxUnavailable` is set too high, and several nodes are cordoned simultaneously, the pod disruption budget (PDB) guarded workloads could fail to drain because a schedulable node cannot be found to run the replicas. If you increase `maxUnavailable` for the machine config pool, ensure that you still have sufficient schedulable nodes to allow PDB guarded workloads to drain.
+
+* Before you begin the upgrade, you must ensure that all the nodes are available. Any unavailable nodes can significantly impact the upgrade duration because the node unavailability affects the `maxUnavailable` and pod disruption budgets.
```

Comment:
@opayne1 I totally agree with you. But this section explains about OCP upgrade duration. To be consistent with the title and overall content, I would suggest:

> Before you being the **cluster** upgrade,.....

wdyt?

Reply:
hmmm honestly I am not sure. I am going to bring @bergerhoffer in on this one for some advice.

Reply:
@bergerhoffer Could you please suggest?

Reply:
@darshan-nagaraj @opayne1 In this case, I would take a look at the whole section of "upgrade duration" and check with the SMEs on whether that can be changed to "update duration", so that it is consistent with what we prefer in OpenShift docs.

We use "update" whenever possible, so I would think that changing this term here to "update" makes sense. For example, we talk about them as "updates" in our general understanding updates section here: http://file.rdu.redhat.com/dnagaraj/OSDOCS-3300/updating/understanding-openshift-updates.html

@darshan-nagaraj Can you check with your SMEs on whether we can make this change? Both Lala and Scott Dodson had approved us preferring the term "update" (in this PR here: https://github.com/openshift/openshift-docs/pull/40525)

Reply:
@sdodson Fixed.